### ISSUE TITLE
Naming issue in checkpoint files

### ISSUE THREAD

**Issue by IrishMehta** [kerner-lab/Mars-Bench#52]
Seeing a potential error in naming checkpoint files. When trying to save the best three files during training, these are the output file names:

1. 'best-epoch=04-val_loss=0.0000.ckpt'  
2. 'best-epoch=07-val_loss=0.0000.ckpt'  
3. 'best-epoch=09-val_loss=0.0000.ckpt'   
4. last.ckpt`

The val-loss is followed by 0.0000 for every checkpoint, which could be due to using some default values instead of the actual values.
